environment.yml

```diff
@@ -3,7 +3,7 @@ name: pandas-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
+  - python=3.10
```

Comment:
might be responsible for some typing failures? perhaps this change is best left to a separate pr

Reply:
Ah, didn't think of that, yes will leave it for another PR then